src/annotator/components/Toolbar.tsx

```diff
@@ -29,18 +29,18 @@ type ToolbarButtonProps = PresentationalProps &
  */
 function ToolbarButton({ icon: Icon, ...buttonProps }: ToolbarButtonProps) {
   return (
-    <ButtonBase
+    <Button
```

Comment:
There's long been a TODO in this module to extract this button variant. It might make sense to add a `toolbar` variant to `IconButton` at some point soon.